glotaran/parameter/test/test_parameter.py

```diff
@@ -1,5 +1,9 @@
+import pickle
```

Comment:
We have considered the security implication of using pickle here (for testing only, in a safe way).